docs/releasenotes/3.1.0.rst

```diff
@@ -75,6 +75,12 @@ New W0918 ``multiline-inline-if`` rule that detects when inline IF is split into
 multiple lines. It's allowed, but it should be avoided, since it decreases readability.
 It recommends to use normal IF/ELSE instead.
 
+New rules that reports empty Template and Test Template (#443)
+--------------------------------------------------------------
+
+Added new W0530 ``empty-template`` rule that reports ``[Template]`` setting without value.
+There is also new E0539 ``empty-test-template`` that reports empty ``Test Template``.
```

Comment:
```suggestion
There is also new E0529 ``empty-test-template`` that reports empty ``Test Template``.
```

Reply:
Maybe let's also explain here why one has a warning level, and another one has error level of severity?

Reply:
Yeah, I will do it